contracts/tokenomics/xastro_token/src/contract.rs

```diff
@@ -279,6 +279,7 @@ pub fn execute(
             marketing,
         } => execute_update_marketing(deps, env, info, project, description, marketing),
         ExecuteMsg::UploadLogo(logo) => execute_upload_logo(deps, env, info, logo),
+        _ => Err(ContractError::Unauthorized {}),
```

Comment:
Why? Without this line we receive more descriptive error in runtime. I suggest to remove this line.

Reply:
Another method was added and we have to ignore it